conf/messages.cy

```diff
@@ -2761,7 +2762,7 @@ error.rentPayableDetails.required = Please give details of selected items includ
 error.rentPayableDetails.maxLength = Description of selected items included in rent payable must be {0} characters or fewer
 
 # WHAT YOU‘LL NEED TO COMPLETE THIS SECTION
-whatYouWillNeed.heading = What you‘ll need to complete this section
+whatYouWillNeed.heading = What you'’'ll need to complete this section
```

Comment:
You just need proper curly apostrophe - ’. Like here:
```includedInRent6020.heading = A yw’r rhent sy’n daladwy yn cynnwys neu’n ymwneud ag unrhyw un o’r canlynol?```
There are no need to enclose it in extra quotation marks